src/extension.ts

```diff
@@ -54,13 +56,15 @@ function registerCommands(context: vscode.ExtensionContext): void {
     const createSkill: CreateSkillWebview = new CreateSkillWebview('Create new skill', 'createSkill', context);
 
     registerWebviews(profileManager, createSkill);
-
-    apiRegisterCommands(context, [
+    ext.askGeneralCommands = [
         new ListSkillsCommand(), new OpenWorkspaceCommand(), new OpenUrlCommand(),
         new InitCommand(profileManager), new GetToolkitInfoCommand(),
         new ViewAllSkillsCommand(), new CreateSkillCommand(createSkill),
         new CloneSkillCommand(), new ChangeProfileCommand(), new AccessTokenCommand(),
-        new DebugAdapterPathCommand(), new CloneSkillFromConsoleCommand()]);
+        new DebugAdapterPathCommand()
```

Comment:
Could you check why delete `CloneSkillFromConsoleCommand` here?

Reply:
Guess this PR needs a rebase from the master, to pick up the latest changes.

Reply:
Yeah, will rebase